utils.c

```diff
@@ -55,6 +55,19 @@ die (const char *format, ...)
 }
 
 void
+die_unless_label_valid (const char *label)
```

Comment:
Should have `__attribute__ ((noreturn))` - see https://github.com/projectatomic/bubblewrap/pull/51